.github/workflows/switch.yml

```diff
@@ -0,0 +1,112 @@
+name: Switch workflow branches
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: "Version to check"
+        required: false
+
+env:
+  NEXT_TAG: ${{ github.event.inputs.version }}
+
+jobs:
+  branch-switch:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/checkout@v3
+        with:
+          path: '/tmp/gitpod/'
+          repository: 'gitpod-io/gitpod'
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+      - name: Install dependencies
+        run: npm i
```

Comment:
Does this work? Since we don't have a package.json I think we need to install dependencies explicitly

Reply:
🤦😆 good catch!